core/src/banking_stage/vote_worker.rs

```diff
@@ -36,8 +36,10 @@ use {
     },
 };
 
-// Step-size set to be 64, equal to the maximum batch/entry size.
-pub const UNPROCESSED_BUFFER_STEP_SIZE: usize = 64;
+// This vote batch size was selected to balance the following two things:
+// 1. Amortize execution overhead (Larger is better)
+// 2. Constrain max entry size for FEC set packing (Smaller is better)
+pub const UNPROCESSED_BUFFER_STEP_SIZE: usize = 16;
```

Comment:
Is there a specific reason we want to do this for votes only? Even if the greedy scheduler doesn't normally batch as much as often, should we still set our max-size **target** to a size of 16?

Reply:
no reason not to that I'm aware of... other than the fact I haven't actually collected performance data on it